Formula/y/yass-cli.rb

```diff
@@ -0,0 +1,148 @@
+class YassCli < Formula
+  desc "Lightweight and efficient, socks/http forward proxy"
+  homepage "https://letshack.info"
+  url "https://github.com/Chilledheart/yass/releases/download/1.13.2/yass-1.13.2.tar.zst"
+  sha256 "fae2f5432e1995087b36240f2c467ab70e5ca3a040b42dd34c383d27fe8388ad"
+  license "GPL-2.0-only"
+  head "https://github.com/Chilledheart/yass.git", branch: "develop"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+  depends_on "nlohmann-json" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "c-ares"
+  depends_on "libnghttp2"
+  depends_on "mbedtls"
+
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "llvm" => :build if DevelopmentTools.clang_build_version < 1500
+  end
+
+  on_linux do
+    depends_on "gperftools"
+  end
+
+  def install
+    ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version < 1500)
+
+    args = %w[
+      -DBUILD_SHARED_LIBS=off
+      -DUSE_BUILTIN_CA_BUNDLE_CRT=off
+      -DCLI=on
+      -DSERVER=on
+      -DGUI=off
+      -DBUILD_TESTS=off
+      -DUSE_SYSTEM_MBEDTLS=on
+      -DUSE_ZLIB=on
+      -DUSE_SYSTEM_ZLIB=on
+      -DUSE_CARES=on
+      -DUSE_SYSTEM_CARES=on
+      -DUSE_SYSTEM_NGHTTP2=on
+      -DUSE_SYSTEM_JSON=on
+    ]
+
+    args << "-DUSE_LIBCXX=on" if OS.mac?
+
+    if OS.linux?
+      args << "-DUSE_TCMALLOC=on"
+      args << "-DUSE_SYSTEM_TCMALLOC=on"
+    end
+
+    system "cmake", "-S", ".", "-B", "build", *(std_cmake_args + args)
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
```

Reply:
Rejected due to no explicit reason.

Reply:
Your reason is hard-code rule. It is not mine. As I stated previous, the std_cmake_cargs messes or will mess with args and that will make failures in building or runtime. State your reason clearly, that's what I think of homebrew as an OSS project.